### ISSUE TITLE
Final accuracy

### ISSUE THREAD

**Issue by bkj** [BoyuanJiang/matching-networks-pytorch#3]
Do you know what the final accuracies of this model trained on Omniglot are?

The results from the paper are:
```
5-way/1-shot  = 98.1%
5-way/5-shot  = 98.9%
20-way/1-shot = 93.8%
20-way/5-shot = 98.5%
```

Thanks
Ben
